lib/cmd/fh3/app/stage.js

```diff
@@ -19,12 +19,23 @@ base.describe['gitRef.value'] = i18n._("The name of the branch you'd like to dep
 base.alias.runtime = "r";
 base.alias.clean = "c";
 base.alias.autodeploy = "d";
+
+//Add custom example
+base.examples[0].cmd = 'fhc app stage --app=<appGuid> --env=<environmentName> (Defaults to branch:master)'
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)